docassemble/ILAO/data/questions/shared-basic-questions.yml

```
@@ -362,4 +357,13 @@ generic object: ALDocumentBundle
 template: x.get_email_copy
 content: |
   Send a copy of your forms to your email
----
\ No newline at end of file
+---
+generic object: ALAddress
+template: x.unit_label
+content: |
+  Address line 2
+---
+generic object: ALAddress
+template: x.postal_code_label
+content: |
+  ZIP Code
```

```suggestion
  ZIP code
```

At the suggestion of @mnewsted .